compiler.ts

```diff
@@ -209,27 +262,27 @@ function codeGenValue(val: Value<Type>, env: GlobalEnv): Array<string> {
 function codeGenBinOp(op : BinOp) : string {
   switch(op) {
     case BinOp.Plus:
-      return "(i32.add)"
+      return "add"
```

Comment:
Why not make these `(call $add)`, `(call $mul)`, etc?

Reply:
Also, what if a user defines a function called `mul`?

Reply:
Sure, thanks for the recommendations! For the binary operations that interact with bignums in codeGenBinOp function, it is now modified to return in the format of `call $$${binop_name}`. For example, for add operation, it will be like `call $$add`. Also, we modified the function name of binary operation from `add` to `$add`, so it is distinguishable from a function call `add`.